lms/djangoapps/course_home_api/progress/v1/serializers.py

```diff
@@ -70,6 +70,7 @@ class CertificateDataSerializer(serializers.Serializer):
     cert_status = serializers.CharField()
     cert_web_view_url = serializers.CharField()
     download_url = serializers.CharField()
+    certificate_available_date = serializers.DateTimeField()
```

Comment:
Nit: alphabetize this... which I think means it goes at the top..

Reply:
Also, wait, you're using this field on the courseware side of things for course exit too.. Did you not need to do any update to the serializer there?